etc/picongpu/hypnos-hzdr/k80_restart.tpl

```diff
@@ -142,7 +143,7 @@ fi
 
 mpiexec --prefix $MPIHOME -x LIBRARY_PATH -npernode !TBG_gpusPerNode -n !TBG_tasks /usr/bin/env bash -c "killall -9 picongpu 2>/dev/null || true"
 
-if [ $endTime -lt $finalStep ]
+if [ $nextStep -lt $finalStep ]
```

Comment:
question:
why the renaming?

Reply:
looks proper to me now. it's the next step that is compared here

Reply:
*offline* discussion with @wmles:
both namings have their pro and cons (`Step`sound more like the next PIC iteration while `$endTime` might be interpreted as final step of all simulations.) Both are fine with me.